src/flutter/shell/platform/linux_embedded/window/native_window.h

```diff
@@ -15,23 +15,41 @@ class NativeWindow {
 
   bool IsValid() const { return valid_; };
 
-#if defined(DISPLAY_BACKEND_TYPE_X11)
+#if defined(DISPLAY_BACKEND_TYPE_X11) || \
```

Comment:
This case is the same with ` EGLNativeDisplayType`. Please consider to replace it with `NativeWindowType`.

Reply:
I replaced it with `EGLNativeWindowType`.